src/types/fantom/fantom.ts

```diff
@@ -0,0 +1,235 @@
+import { CryptoNodeData, VersionDockerImage } from '../../interfaces/crypto-node';
+import { defaultDockerNetwork, NetworkType, NodeClient, NodeType, Role, Status } from '../../constants';
+import { Docker } from '../../util/docker';
+import { ChildProcess } from 'child_process';
+import { v4 as uuid} from 'uuid';
+import path from 'path';
+import os from 'os';
+import { Ethereum } from '../ethereum/ethereum';
+import { filterVersionsByNetworkType } from '../../util';
+import { FS } from '../../util/fs';
+import * as coreConfig from './config/core';
+
+
+export class Fantom extends Ethereum {
+
+  static versions(client: string, networkType: string): VersionDockerImage[] {
+    client = client || Fantom.clients[0];
+    let versions: VersionDockerImage[];
+    switch(client) {
+      case NodeClient.CORE:
+        versions = [
+          {
+            version: '1.1.0-rc.5',
+            clientVersion: '1.1.0-rc.5',
+            image: 'chainstack/fantom-opera:1.1.0-rc.5',
```

Comment:
I went ahead and built our own docker images so that we don't have to wait for chainstack for updates. I added the current version you used as well as the newer version which there isn't yet a chainstack image for. After you have updated `1.1.0-rc.5` and verified that everything works as expected, you can add `1.1.1-rc.1` as another version.

rburgett/fantom-opera:v1.1.0-rc.5
rburgett/fantom-opera:v1.1.1-rc.1

Reply:
The old genesis file won't work with v1.1.1-rc.1. Either a node needs to have already been initiated or we can try the one below:

https://download.fantom.network/mainnet-109331-pruned-mpt.g

Reply:
72GB?!?!?!?!?! How is that even possible. 😑